usaspending_api/recipient/v2/views/list_recipients.py

```diff
@@ -70,13 +71,59 @@ def get_recipients(filters={}):
     return results, page_metadata
 
 
+def duns_count_key_function(view_instance, view_method, request, args, kwargs):
+    validated_payload = TinyShield(view_instance.models).block(request.data)
+
+    award_type = "-"
+    if "award_type" in validated_payload:
+        award_type = validated_payload["award_type"]
+
+    keyword = "-"
+    if "keyword" in validated_payload:
+        keyword = validated_payload["keyword"]
+
+    return ".".join(
+        [
+            award_type,
+            keyword,
+        ]
+    )
+
+
+class DunsCount(APIView):
```

Comment:
The think `RecipientCount` is more agreeable. I assume there will be many recipients without an associated DUNS returned in the count

Reply:
Agreed. I updated this. I left them in the same file though, so they can continue to share methods without circular dependencies